README.md

```diff
@@ -0,0 +1,66 @@
+<img src="/doc-base/neo.png" align="right" height="70"/>
+
+# NEO Specification
+
+## document format
+The Community Specification Paper base document started as pure LaTeX (as a [Yellow Paper](https://github.com/NeoResearch/yellowpaper])), as usual in academic publications, however the syntax is not considered as friendly as Markdown. So, why not merge both? :)
```

Comment:
Remove this line?

Reply:
As you want, maybe it is a good introduction.
However, maybe not necessary.

What do you think @igormcoelho?